app/controllers/data.js

```diff
@@ -49,242 +49,233 @@ var DataController = Ember.ArrayController.extend({
       Ember.$('#alert-box').fadeOut(300);
     }, 5000);
   },
-  resetSelection: function() {
+  resetSelection() {
     this.set('selectedCollection', null);
     this.set('selectedCommand', null);
     this.set('executedCommand', false);
     this.set('commandResults', null);
     this.set('jsonError', null);
   },
   actions: {
-    selectCollection: function(collectionName) {
-      Ember.$('#' + collectionName).parents('.btn-group').find('.dropdown-toggle').html(collectionName + ' <span class="caret"></span>');
+    selectCollection(collectionName) {
+      Ember.$(`#${collectionName}`).parents('.btn-group').find('.dropdown-toggle').html(`${collectionName} <span class="caret"></span>`);
       this.set('selectedCollection', collectionName);
       this.set('executedCommand', false);
       this.set('commandResults', null);
       this.set('jsonError', null);
     },
-    selectCommand: function(command) {
-      Ember.$('#command-' + command).parents('.btn-group').find('.dropdown-toggle').html(command + ' <span class="caret"></span>');
+    selectCommand(command) {
+      Ember.$(`#command-${command}`).parents('.btn-group').find('.dropdown-toggle').html(`${command} <span class="caret"></span>`);
       this.set('selectedCommand', command);
       this.set('executedCommand', false);
       this.set('commandResults', null);
       this.set('jsonError', null);
     },
-    showOptions: function() {
+    showOptions() {
       this.set('showCommandOptions', !this.get('showCommandOptions'));
     },
-    submitCommand: function() {
+    submitCommand() {
       this.set('executedCommand', false);
       this.set('commandResults', null);
       this.set('jsonError', null);
 
-      var requestBody = { options: {} };
+      const requestBody = { options: {} };
 
       if (this.get('showQueryInput') && this.get('searchQueryText') && this.get('searchQueryText').length > 0) {
-        var jsonQuery;
+        let jsonQuery;
         try {
           jsonQuery = JSON.parse(this.get('searchQueryText'));
-        }
-        catch (e) {
-          this.set('jsonError', 'Error parsing query: ' + e);
+        } catch (e) {
+          this.set('jsonError', `Error parsing query: ${e}`);
           return;
         }
 
         requestBody.query = jsonQuery;
       }
 
       if (this.get('showJSONInput') && this.get('entityText') && this.get('entityText').length > 0) {
-        var entity;
+        let entity;
         try {
           entity = JSON.parse(this.get('entityText'));
-        }
-        catch (e) {
-          this.set('jsonError', 'Error parsing JSON: ' + e);
+        } catch (e) {
+          this.set('jsonError', `Error parsing JSON: ${e}`);
           return;
         }
 
         requestBody.entity = entity;
       }
 
-      var option, optionEnabled, optionInputValue;
-      var optionsArray = this.get('selectedCommandOptions');
-      for (var i=0; i < optionsArray.length; i++) {
+      let option; let optionEnabled; let
```

Comment:
Each of these should be in its own line (same for any other occurrence of this).

Reply:
Fixed.

Reply:
This changed seems to have never happened (or been reverted by mistake)

Reply:
Yep, seems I forgot to commit the change. Sorry about it.